doc/api/repl.markdown

```diff
@@ -190,15 +194,59 @@ be emitted.
 Example of listening for `reset`:
 
     // Extend the initial repl context.
-    var r = repl.start({ options ... });
+    var replServer = repl.start({ options ... });
```

Comment:
Other pre-existing examples in the doc use `var` for such things, so I thought it was best to keep with the style already in this doc. Perhaps instead a future PR can convert them all in one fell swoop?